packages-next/auth/src/components/Icons.tsx

```diff
@@ -0,0 +1,81 @@
+/* @jsx jsx */
+
+import { forwardRefWithAs, jsx } from '@keystone-ui/core';
+
+const Icon: React.ComponentType = ({ children, ...props }) => {
```

Comment:
pls don't use `ComponentType`, type the props instead since `ComponentType` includes children as a prop always(which sure, is correct in this case but it's wrong generally and someone reading it probably won't know that `ComponentType` includes `children`). I'll make a PR enforcing that it's not used with ESLint.

Reply:
sure thing, no idea that was a rule for us 👍